code/modules/jobs/job_types/prisoner.dm

```diff
@@ -54,6 +54,9 @@
 	monkestation end */
 	to_chat(crewmember, span_warning("You are imprisoned for \"[crime_name]\"."))
 	crewmember.add_mob_memory(/datum/memory/key/permabrig_crimes, crimes = crime_name)
+	var/perpname = crewmember.real_name
+	var/datum/record/crew/jailbird_record = find_record(perpname)
+	jailbird_record.wanted_status = WANTED_PRISONER
```

Comment:
this is runtiming. and minds store a ref to their crew+locked records anyways, so you can just do this
```suggestion
	crewmember.mind?.crewfile?.wanted_status = WANTED_PRISONER
```